frontend/testing/requests/feed.test.js

```
@@ -68,6 +68,8 @@ describe("Feed tests", () => {
         },
       ];
 
+      const mockDroplet = { id: 10, name: "Sample Droplet" };
```

ESLint

[error] 'mockDroplet' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)